src/audiences/edit/data/index.js

```diff
@@ -0,0 +1,37 @@
+const { apiFetch } = wp;
+
+export const getEstimate = async ( audience ) => {
+	try {
+		const audienceQuery = encodeURIComponent( JSON.stringify( audience ) );
+		// Get audience estimate data.
+		const data = await apiFetch( {
+			path: `analytics/v1/audiences/estimate?audience=${ audienceQuery }`,
+		} );
+
+		return data;
+	} catch ( error ) {
+		return { error };
+	}
```

Comment:
I don't love these; could we instead do the catching in the caller?

Reply:
Seems like a lot of duplication to do things that way around?

Reply:
Updated 👍